test/controllers/api/v2/roles_controller_test.rb

```diff
@@ -60,18 +60,16 @@ class Api::V2::RolesControllerTest < ActionController::TestCase
     assert_response :success
   end
 
-  test "should clone role and its taxonomies" do
+  test "should clone role without taxonomies" do
```

Comment:
I think it makes a lot of sense from a UX perspective to also clone the taxonomies. Why do you want to remove this feature?

Reply:
We want to unify process of cloning roles because in UI when you clone role it will be cloned without taxonomies and in CLI it clone with taxonomies(if you want to update role's taxonomies you will override cloned taxonomies) . Now in CLI when you clone role it will be cloned without taxonomies and you can easily add them by --organization-ids/--location_ids. It make sense to not have same roles in same locations or organizations.

Reply:
I agree with @timogoebel here, clone imply that it's a copy. We should solve this exactly the other way around - modifying the UI to copy taxonomy for the clone. That one will be probably more challenging than this solution, but it is worth the effort I think.

This also breaks existing API contract, this is my second argument against this.

Reply:
Ok I will look at that